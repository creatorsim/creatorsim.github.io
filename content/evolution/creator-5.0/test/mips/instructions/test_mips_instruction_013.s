
#
# Creator (https://creatorsim.github.io/creator/)
#

.data
    w1:		.word 14
    b1:		.byte 127

    .align 1
    h1:		.half 32767
    w2:		.space 4
    b2:		.space 1

    .align 1
    h2:		.space 2

.text
main:

    la $t0, w1
    la $t1, b1
    la $t2, h1

    lw $t3, 0($t0)
    lw $t4, w1

    lb $t5,  0($t1)
    lbu $t6, b1

    lh $t7,  0($t2)
    lhu $t8, h1

    la $t9, w2
    sw $t3, 0($t9)

    la $t9, b2
    sb $t5, 0($t9)

    la $t9, h2
    sh $t7, 0($t9)	
