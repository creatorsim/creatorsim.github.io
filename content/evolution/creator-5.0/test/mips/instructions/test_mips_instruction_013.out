
cr[PC]:0x84; ir[1,at]:0x20000e; ir[8,t0]:0x200000; ir[9,t1]:0x200004; ir[10,t2]:0x200006; ir[11,t3]:0xe; ir[12,t4]:0xe; ir[13,t5]:0x7f; ir[14,t6]:0x7f; ir[15,t7]:0x7fff; ir[24,t8]:0x7fff; ir[25,t9]:0x20000e; memory[0x20000f]:0xff; memory[0x20000e]:0x7f; memory[0x20000c]:0x7f; memory[0x20000b]:0x0e; keyboard[0x0]:''; display[0x0]:''; 

